frontend/src/components/Navbar.tsx

```
@@ -7,6 +7,13 @@ import { useRouter } from "next/navigation";
 import Image from "next/image";
 import axios from "axios";
 import Cookies from "js-cookie";
+import { MagnifyingGlassIcon } from "@heroicons/react/24/outline";
+
+// Local Files (Import)
+import { useAuth } from "../hooks/AuthProvider";
```

Biome (1.9.4)

[error] Shouldn't redeclare 'useAuth'. Consider to delete it or rename it.

'useAuth' is defined here:


(lint/suspicious/noRedeclare)